### DISCUSSION TITLE
Connecting to multiple AMQP brokers

### DISCUSSION THREAD

**Discussion by Kreczat** [roadrunner-server/roadrunner#1912] [Q&A] [answered]
Hello.
I need to connect several AMQP brokers in one project (the first broker is used for data receiving from external service, the second is for cross-microservice interaction. As far as I understand, rr.yaml config allow establish only one connection per one message broker kind. Is there a way to establish more than one connection per one message broker kind (AMQP for me)?

**Comment by rustatian** [comment]
Hey @Kreczat 👋 
You may start several RR instances, this is the only option at the moment.

**Reply by Kreczat** [reply]
Is it possible to add this feature to one of next releases?

**Reply by rustatian** [reply, marked as answer]
Not sure that this is a wide use-case, but still, you may create a feature request, describe how you see this feature, config samples, etc. Since this is an addition, w/o breaking changes, you may even send us a PR 😃 
If this FR will be popular, definitely it'll be included in one of the future releases. Keep in mind, that if you need this functionality right now, you may fork this plugin, implement this feature for yourself and build RR with our tool called Velox.

**Reply by shieldz80** [reply]
Hi @rustatian,

Do you know if a FR was created for this? I can't seem to find one. I also have a similar use case in my projects at work where I need to connect to multiple brokers.

**Reply by Kreczat** [reply]
Hi, @shieldz80 

I didn't create any FR for this, because I've found any solution for my purposes. But if you create such FR, I will like it.

**Reply by shieldz80** [reply]
Thanks for the info @Kreczat, I will create a FR then as soon as possible.

**Reply by shieldz80** [reply]
@Kreczat Created https://github.com/roadrunner-server/roadrunner/issues/1962
